tests/Doctrine/Tests/Models/AivusTest/Category.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Doctrine\Tests\Models\AivusTest;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+
+/**
+ * @Entity
+ * @Table(name="categories")
+ */
+class Category
+{
+    /**
+     * @Column(type="integer")
+     * @Id
+     * @GeneratedValue
+     *
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @ManyToMany(targetEntity="Product")
+     * @JoinTable(name="category_to_products",
+     *      joinColumns={@JoinColumn(name="category_id", referencedColumnName="id")},
+     *      inverseJoinColumns={@JoinColumn(name="product_id", referencedColumnName="id", unique=true)}
```

Comment:
This is not valid, as by definition a many-to-many association allows many of both sides. This is a `one-to-many` association that may be implemented incorrectly.